### PR TITLE
Rename folders

### PR DESCRIPTION
**Pull request by audiologic-vannoy**
Folders are now snake_case, and the Source_Files folders are gone.
Updated gitignore to ignore generated files

If there are any Quartus projects that explicitly have any of these directories in their library search path, those projects are likely now broken and need to be updated.

### CODE REVIEW

File: ad1939/.tmp_versions/FE_AD1939.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/mnt/data/NIH/component_library/AD1939/Source_Files/FE_AD1939.ko
```
Comment (fe-wickham) [inline ad1939/.tmp_versions/FE_AD1939.mod]: Remove file

File: qsys_simple_ha/.tmp_versions/FE_Qsys_Simple_HAv8.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/mnt/data/NIH/component_library/Qsys_Simple_HAv8/Source_Files/FE_Qsys_Simple_HAv8.ko
```
Comment (fe-wickham) [inline qsys_simple_ha/.tmp_versions/FE_Qsys_Simple_HAv8.mod]: Remove file

**Review by fe-wickham — CHANGES_REQUESTED** [review]
Mostly looks good. Some .mod files did get added in, probably before the .gitignore. Additionally, should the AD1939 pdf docs be deleted?

**Comment by tvannoy** [comment]
> should the AD1939 pdf docs be deleted?

I don't think there's much reason to have external datasheets in our version control. I think we should include links to them in our READMEs, though.

**Comment by rksnider** [comment]
Links would be fine.

**Review by fe-wickham — APPROVED** [review]
Looks good

**Pull request merged**